ui/src/main/resources/FileManagerCode/DriveSheet.xml

```diff
@@ -133,7 +133,7 @@
   #set ($jobId = $services.drive.pack($paths, $packReference))
   #if ($jobId)
     #set ($discard = $downloadDoc.set('jobId', $jobId))
-    #if ($xwiki.checkAccess($downloadDoc, "edit"))
+    #if ($xwiki.hasAccessLevel("edit", $xcontext.getUser() , $downloadDoc))
```

Comment:
Read again my comment :)

> Better use http://maven.xwiki.org/site/docs/xwiki-javadoc-4.1.x/com/xpn/xwiki/api/Document.html#hasAccessLevel(java.lang.String) .

Don't you think the method I indicated is simpler?